apps/meteor/client/components/MarkdownText.tsx

```diff
@@ -109,7 +110,14 @@ const MarkdownText: FC<Partial<MarkdownTextParams>> = ({
 			}
 		})();
 
-		return preserveHtml ? html : html && sanitizer(html, { ADD_ATTR: ['target'] });
+		const URIschemes = new RegExp(
+			'^(?:(?:(?:f|ht)tps?|mailto|tel|callto|cid|xmpp|' +
+				`${settings.get('Markdown_URIAllowedSchemes')}` +
```

Comment:
```suggestion
				`${settings.get('Markdown_SupportSchemesForLink').split(',').join('|')}` +
```

I don't think we should create a new preference just for this case... We could use the `Markdown_SupportSchemesForLink` setting.